ocs_ci/ocs/cluster.py

```diff
@@ -189,7 +189,7 @@ def is_health_ok(self):
         self.cluster.reload()
         return self.cluster.data['status']['ceph']['health'] == "HEALTH_OK"
 
-    def cluster_health_check(self, timeout=None):
+    def cluster_health_check(self, mcg_obj_session, timeout=None):
```

Comment:
is `mcg_obj_session` a fixture?
if so, this usage is wrong as cluster_health_check is not a test function. Pytest recognized this usage only with a function used for the test (i.e, `test_`)

Reply:
changed to initialize an MCG object within the class